connectivity/drivers/ble/FEATURE_BLE/COMPONENT_BlueNRG_MS/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-target_sources(mbed-os
-    PRIVATE
+add_library(mbed-os-ble-bluenrg INTERFACE)
```

Comment:
```suggestion
add_library(mbed-os-ble-blue_nrg INTERFACE)
```

Reply:
as the original name is BlueRNG, isn't it better to be just bluenrg (using lowercase always for target names).

Reply:
We changed it from using CamelCase to snake_case.

Reply:
Althought I dont like mixing `-` with `_`. Either it would be separated by `-` or all together. But no strong preference here, I'll fix.